packages/template-retail-react-app/app/components/footer/index.jsx

```diff
@@ -208,7 +205,10 @@ const Subscribe = ({...otherProps}) => {
 
             <Box>
                 <InputGroup>
-                    <Input type="email" placeholder="you@email.com" {...styles.subscribeField} />
+                    {/* Had to swap the following InputRightElement and Input 
+                        to avoid the hydration error due to mismatched html between server and client side.
+                        This is a workaround for Lastpass plugin that automatically injects its icon for input fields.
```

Comment:
Lastpass plugin cause hydration mismatch warnings? That's very interesting, I'd imagine browser plugins run after hydration 🤷

Reply:
Yes, it is also very intermittent issue, the hydration error would appear if the plugins are interfering with the DOM during hydration

Reply:
@kevinxh other people have similar issue with some browser plugins too: https://github.com/facebook/react/issues/24430